gating/pre_merge_test/run_deploy.sh

```diff
@@ -41,6 +41,11 @@ elif [[ "${RE_JOB_SCENARIO}" == "ironic" ]]; then
   export DEPLOY_IRONIC="yes"
 fi
 
+# Set the apt artifact mode
+if [[ ${RE_JOB_IMAGE} =~ loose$ ]]; then
+  export RPCO_APT_ARTIFACTS_MODE="loose"
```

Comment:
Add an upgrade of all packages here so that we have the absolute latest packages installed.